### PR TITLE
:lipstick: Add border style to website 

### PR DESCRIPTION
**Pull request by jeyemwey**
This PR:
* adds a border style to the website 
* only shows the borderColor property if not null
* and adds the color blip to the borderColor property when given

Looks like this in the website:

image

### CODE REVIEW

**Comment by MrKrisKrisu** [comment]
Looks good!
Is this the same as #25?

File: website/index.php [hunk 1]
```diff
@@ -116,9 +116,10 @@ class="w-75"></span><span class="w-50"></span></button>
 
                                 <dl>
                                     <?php foreach ($line as $key => $value): ?>
+                                        <?php if ($key === "borderColor" && $value === "") continue; ?>
```
Comment (MrKrisKrisu) [inline website/index.php:119]: I think, shortOperatorName and lineName can also be skipped.

Comment (jeyemwey) [reply website/index.php:119]: At this point, we could maybe also move to a white-list of properties that we want to show.

**Comment by jeyemwey** [comment]
> Is this the same as https://github.com/Traewelling/line-colors/pull/25?

Yes mainly, added some details (i.e. the borderColor color blip and skipping if not present) and made it with `em` instead of `px`.

File: website/index.php (hunk 1 above)
Comment (MrKrisKrisu) [reply website/index.php:119]: Makes also sense.

But shortOperatorName is the same in every section as the heading and the lineName is rendered in the icon :D

**Review by marhei — APPROVED** [review]

**Pull request merged**